config/initializers/refinery/core.rb

```diff
@@ -31,7 +31,7 @@
 
   # Specify a different backend path than the default of "refinery".
   # Make sure you clear the `tmp/cache` directory after changing this setting.
-  # config.backend_route = "refinery"
+  config.backend_route = "hubadmin"
```

Comment:
Hub admin makes sense now, but the site might not be called the digital hub forever. Could I suggest that the backend route simply be /admin ?

Reply:
Using endpoints like `/admin` will set us up for zero day attacks. I agree that `/hubadmin` might be too specific but I think that `/admin` is too generic. 

How about something like `/staff` or `/internal`?